packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2078,105 +2047,6 @@ void main() {
     );
   });
 
-  testUsingContext(
```

Comment:
I don't think you want to delete these two tests since they are showing directories can be regenerated without Swift Package Manager, that's still an important test. The iOS language wasn't particularly relevant. 
You'll have to update the expectation from 
```diff
- 'ios/Classes/FlutterProjectPlugin.h',
+ 'ios/Classes/FlutterProjectPlugin.swift',
```